standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/HMSHandler.java

```diff
@@ -608,34 +549,21 @@ private void authorizeTableForPartitionMetadata(
         isServerFilterEnabled, filterHook, catName, dbName, tblName);
   }
 
-  private static String addPrefix(String s) {
```

Comment:
it seems like log messages were prefixed with the `threadLocalId` - which is being removed here; or its being retained differently?
* its already displayed in the name of the thread..
* or it doesnt really add much value as the thread's name should already be in the logs...
history takes me back HIVE-1818 and HIVE-1081 ; so the original intention is not that clear

Reply:
I see the other thread - lets remove threadid